api/src/pcapi/utils/urls.py

```diff
@@ -41,11 +41,11 @@ def build_pc_pro_offer_link(offer: CollectiveOffer | CollectiveOfferTemplate | O
 
 
 def build_pc_pro_offerer_link(offerer: offerers_models.Offerer) -> str:
-    return f"{settings.PRO_URL}/accueil?structure={offerer.id}"
+    return f"{settings.PRO_URL}/accueil?structure={offerer.id}&frombo=true"
```

Comment:
```suggestion
    return f"{settings.PRO_URL}/accueil?structure={offerer.id}&from-bo=true"
```

Il y a un tiret dans https://github.com/pass-culture/pass-culture-main/blob/8de00c8348537e9389a2920acd17b29296d8d567/pro/src/app/App/App.tsx#L52